.github/workflows/test_rsync_test_script.yml

```
@@ -0,0 +1,47 @@
+name: test rsync check
+
+on:
+  pull_request:
+    paths:
+      - 'metomi/rose/loc_handlers/rsync.py'
+      - 'metomi/rose/loc_handlers/rsync_remote_check.py'
```

I think we might need to list the action itself?

```suggestion
      - 'metomi/rose/loc_handlers/rsync_remote_check.py'
      - '.github/workflows/test_rsync_test_script.yml'
```